.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
I'm not necessarily against having a gitignore in this repo, but maybe it shouldn't come in as an accident of the PR?

I personally configure a user-level gitignore to ignore all the IDE crap, otherwise each project has to have gitignores for every IDE in existence...

Reply:
Roger that. I will take this file out.